security/pkg/nodeagent/util/util.go

```diff
@@ -25,8 +25,12 @@ import (
 	"go.opencensus.io/stats/view"
 
 	"istio.io/istio/pkg/file"
+	"istio.io/pkg/env"
 )
 
+var k8sInCluster = env.RegisterStringVar("KUBERNETES_SERVICE_HOST", "",
+	"Kuberenetes service host, set automatically when running in-cluster")
```

Comment:
Can you point to the code that sets KUBERNETES_SERVICE_HOST in k8s environment for Istio Agent?

Reply:
kubernetes sets it in all pods